cif_pow.dic

```diff
@@ -4648,6 +4648,647 @@ save_pd_phase.name
+    correction. Valid values are positive, even integers.
+;
+    _name.category_id             pd_pref_orient
+    _name.object_id               sphericalharmonics_y_i
+    _type.purpose                 Number
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Integer
+    _enumeration.range            0:
+    _units.code                   none
+
+save_
+
+save_pd_pref_orient.sphericalharmonics_y_ij
+
+    _definition.id                '_pd_pref_orient.sphericalharmonics_y_ij'
+    _definition.update            2022-09-28
+    _description.text
+;
+    The order (i) and  term (j) of the spherical harmonic
+    preferred-orientation correction. Spherical
+    harmonics are functions that obey the point-symmetry
+    operations of the phase, and can be used to model
+    deviations in peak intensity due to preferred-orientation
+    or texture.
+
+    In the application of spherical harmonics corrections,
+    valid orders are positive, even integers, and possible
+    terms are in the range -i:i. Valid terms are restricted
+    by space-group symmetry. The parity of the term is given
+    by it's sign; odd parity is negative, even is positive.
+
+    Omitting _pd_pref_orient.* implies that no preferred-
+    orientation correction has been used.
+
+    See Jarvinen, M. (1993). J. Appl. Cryst. 26, 525-531 for
+    further information.
+;
+    _name.category_id             pd_pref_orient
+    _name.object_id               sphericalharmonics_y_ijp
+    _type.purpose                 Number
+    _type.source                  Assigned
+    _type.container               Matrix
+    _type.dimension               '[2]'
+    _type.contents                Integer
+    _units.code                   none
+    _method.purpose               Evaluation
+    _method.expression
+;
+    With po as pd_pref_orient
+
+      _pd_pref_orient.sphericalharmonics_y_ij =
+        [po.sphericalharmonics_y_i, po.sphericalharmonics_y_j]
+;
+
+save_
+
+save_pd_pref_orient.sphericalharmonics_y_j
+
+    _definition.id                '_pd_pref_orient.sphericalharmonics_y_j'
+    _definition.update            2022-09-28
+    _description.text
+;
+    The term (j) of the given order (i) of the spherical harmonics
+    preferred-orientation correction. The parity of the term
+    is given by it's sign; odd parity is negative, even is positive.
+
+    In general, possible values are in the range -i:i. Valid values
+    are dependent on the space-group of the phase.
+;
+    _name.category_id             pd_pref_orient
+    _name.object_id               sphericalharmonics_y_j
+    _type.purpose                 Number
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Integer
+    _units.code                   none
+    _method.purpose               Definition
+    _method.expression
+;
+     With po as pd_pref_orient
```

Comment:
I don't know if there is any dREL validation as part of the commit checks, but is this valid?

Reply:
There is no dREL validation check at this point. As far as dREL is concerned I think we can add/correct at our leisure, the important thing at this point is that the intention is clear to the human reader.